drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -3892,17 +3890,8 @@ bool isArrayTexture(MTLTextureType p_type) {
 
 bool RenderingDeviceDriverMetal::has_feature(Features p_feature) {
 	switch (p_feature) {
-		case SUPPORTS_MULTIVIEW:
```

Comment:
@DarioSamo was this by accident? Multiview support was just added to Metal and shouldn't be effected by the VRS work?

Reply:
Never mind, noticed the code had already moved to `RenderingDevice::has_feature`, so this seems to be some cleanup